src/Sparrow/ByteString.cs

```diff
@@ -497,7 +500,7 @@ public ByteStringContext(int allocationBlockSize = DefaultAllocationBlockSizeInB
         { }
     }
 
-    public unsafe class ByteStringContext<TAllocator> : IDisposable where TAllocator : struct, IByteStringAllocator
+    public unsafe class ByteStringContext<TAllocator> : ILowMemoryHandler, IDisposable where TAllocator : struct, IByteStringAllocator
```

Comment:
We already have a context for that, the JsonContextPool, there is no need to also handle it here.
You need to pass the LowLevelFlag from the pool, not maintain your own here.